### DISCUSSION TITLE
8bitUHM Discussions for theta360-automotive-guide

### DISCUSSION THREAD

**Discussion by codetricity** [8bitUHM/theta360-automotive-guide#42] [Announcements]
## 👋 Aloha!
  We’re using Discussions as a place to share technology best practices with Python, Django, SQL, cloud-based services, and related topics. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other members member of 8bitUHM and the stakeholders.

**Comment by jcasman** [comment]
Oh, very friendly. Hello from California, too. 🏄
